README.md

```diff
@@ -67,85 +67,55 @@ Currently supported:
 ```
 - archive/
 - indexes/
+- pages/
 - datapackage.json
 ```
 
 ### Directories and Files
 
- 
+
 #### 1) `archive/` (required)
- 
+
 The archives directory can contain raw web archive data.
 
 Currently supported formats:
 - WARC (.warc, .warc.gz)
 
 #### 2) `indexes/` (required)
- 
+
  The indexes directory should include various indexes into the raw data stored in `archives/`
- 
+
  Currently possible index formats include:
  - CDX (.cdx, .cdxj) for raw text-based binary sorted indices
  - Compressed CDX (.cdx.gz and .idx) indices
- 
+
 #### 3) `datapackage.json` (required)
 
 This file serves as the manifest for the web archive and is compliant with the frictionless (data standard)[https://frictionlessdata.io/data-package/#the-data-package-suite-of-specifications]
 
-The file 
+The file
 
 Keys specified in this file take precedent over other files, and provide a way to override the configuration
 of the web archive.
 
-`resources` is a list containing objects with this structure as a default 
+`resources` is a list containing objects with this structure as a default
 ```{"path": "example_path", "stats": {"hash": "example_hash", "bytes": 0}```
 
-
-
-##### `pages` key
+#### 4) `pages` (optional)
```

Comment:
We decided to make `pages/pages.jsonl` required, though additional *.jsonl files can also be added with the same format.